sct.py

```diff
@@ -193,7 +194,16 @@ def provision_resources(backend, test_name: str, config: str):
 
     add_file_logger()
 
-    layout = SCTProvisionLayout(params=create_sct_configuration(test_name=test_name))
+    params = create_sct_configuration(test_name=test_name)
+    test_config = get_test_config()
+    localhost = LocalHost(user_prefix=params.get("user_prefix"), test_id=test_config.test_id())
+
+    if params.get("logs_transport") == 'rsyslog':
+        test_config.configure_rsyslog(localhost, enable_ngrok=False)
+    elif params.get("logs_transport") == 'syslog-ng':
+        test_config.configure_syslogng(localhost)
```

Comment:
Maybe would be better to add "else" clause where we log "warning" saying that none of service loggers were configured?

Reply:
added